README.md

```diff
@@ -46,9 +46,9 @@ playground for learning and exploring.
 
 ## Rules / Aliases
 
-* **[MD001](doc/Rules.md#md001)** *header-increment* - Header levels should only increment by one level at a time
-* **[MD002](doc/Rules.md#md002)** *first-header-h1* - First header should be a top level header
-* **[MD003](doc/Rules.md#md003)** *header-style* - Header style
+* **[MD001](doc/Rules.md#md001)** *heading-increment, header-increment* - Heading levels should only increment by one level at a time
```

Comment:
There is commented-out code in `markdownlint.js` to generate this. I don’t see that changed here, but it would be good to do for future updates.